.buildkite/commands/run-ui-tests.sh

```diff
@@ -27,4 +27,8 @@ install_cocoapods
 echo "--- 🧪 Testing"
 xcrun simctl list >> /dev/null
 rake mocks &
-bundle exec fastlane test_without_building name:"$TEST_NAME" device:"$DEVICE" ios_version:"$IOS_VERSION"
+bundle exec fastlane test_without_building \
+  name:UITests \
```

Comment:
I decided to hardcode `name:UITest` because, unlike the other parameters, this one is currently the same for all UI tests invocations.